src/lambda/expression.rs

```diff
@@ -157,6 +160,16 @@ impl Eval for Expression {
                 Expression::Logic(logic) => logic.eval(ctx, conc).await,
                 Expression::List(list) => list.eval(ctx, conc).await,
                 Expression::Math(math) => math.eval(ctx, conc).await,
+                Expression::Object(object) => {
+                    match object {
+                        Object::Path(path) => Ok(ctx
+                            .value()
+                            .ok_or(anyhow!("Expected a value for path: {}", object))?
+                            .get_path(path)
+                            .cloned()
+                            .ok_or(anyhow!("No such value exists"))?),
```

Comment:
Don't need to throw an error. We should just return null value if nothing is available.